### ISSUE TITLE
history not have mean_q

### ISSUE THREAD

**Issue by hardy110** [keras-rl/keras-rl#326]
i use duel_dqn_cartpole ,find a problem。
his=dqn.fit(env, nb_steps=1000, visualize=False, verbose=2)
use his.history ，just find   episode_reward , nb_steps and nb_episode_steps 。
his.history
{'episode_reward': [-51.53412198149204, 3.981158740171498], 'nb_steps': [487, 974], 'nb_episode_steps': [487, 487]}
but  log have more info :
487/1000: episode: 1, duration: 377.151s, episode steps: 487, steps per second: 1, episode reward: 32.765, mean reward: 0.067 [-3.768, 7.021], mean action: 0.955 [0.000, 2.000], mean observation: 0.012 [-1.623, 5.089], loss: 1.848188, mean_absolute_error: 22.578606, mean_q: 33.592277
how can i find info about mean_q in his.history ? or other way ? 

Thank you!

- [ ] Check that you are up-to-date with the master branch of Keras-RL. You can update with:
`pip install git+git://github.com/keras-rl/keras-rl.git --upgrade --no-deps`

- [ ] Check that you are up-to-date with the master branch of Keras. You can update with:
`pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps`

- [ ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short). If you report an error, please include the error message and the backtrace.
